<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>768</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1024</width>
      <height>721</height>
     </rect>
    </property>
    <widget class="QWidget" name="page">
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>830</x>
        <y>70</y>
        <width>151</width>
        <height>111</height>
       </rect>
      </property>
      <property name="text">
       <string>검색</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>80</y>
        <width>481</width>
        <height>91</height>
       </rect>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox_1">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>250</y>
        <width>76</width>
        <height>22</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>D</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>F</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>L</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>N</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>P</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Q</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>O</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>R</string>
       </property>
      </item>
     </widget>
     <widget class="QComboBox" name="comboBox_2">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>250</y>
        <width>76</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox_3">
      <property name="geometry">
       <rect>
        <x>760</x>
        <y>250</y>
        <width>76</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QTableWidget" name="tableWidget">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>290</y>
        <width>881</width>
        <height>431</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>250</y>
        <width>80</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>대분류명</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>510</x>
        <y>250</y>
        <width>80</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>중분류명</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>860</x>
        <y>250</y>
        <width>80</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>소분류명</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2"/>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1024</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
